addons/editor-cleanup-plus/addon.json

```diff
@@ -10,6 +10,14 @@
       "link": "https://scratch.mit.edu/users/Chrome_Cat/"
     }
   ],
+  "settings": [
+    {
+      "name": "Find unused variables",
```

Comment:
A setting description might be helpful here (to explain that this shows a prompt and that it also removes reporter blocks).